sklearn/covariance/_robust_covariance.py

```diff
@@ -206,6 +204,10 @@ def _c_step(
             print("Maximum number of iterations reached")
         results = location, covariance, det, support, dist
 
+    location, covariance, det, support, dist = results
+    # Convert from list of indices to boolean mask.
+    support = np.bincount(support, minlength=n_samples).astype(bool)
```

Comment:
I find the following a bit more straightforward
```suggestion
    support = np.isin(np.arange(n_samples), support)
```

Reply:
For e.g. 500 elements in the support and 500 elements out of it, bincount is *much* faster than isin (~10x), likely because the structure of the test elements is much more constrained.

Reply:
good to know, thanks